tools/backups2datalad/adataset.py

```diff
@@ -231,9 +233,6 @@ async def add(self, path: str) -> None:
         await self.call_annex("add", path)
 
     async def remove(self, path: str) -> None:
-        # to avoid problems with locking etc. Same is done in DataLad's invocation
-        # of rm
-        self.ds.repo.precommit()
```

Comment:
hm, not sure on this one -- didn't investigate but if we do use some batched process through datalad interfaces -- we might still need this one... I guess we will see